.gitlab-ci.yml

```diff
@@ -57,5 +57,41 @@ deploy-python-tool-production:
     ENVIRONMENT_NAME: production
     EVA_SUBMISSION_PATH: $EVA_SUBMISSION_PRODUCTION_PATH
     DEPLOY_POINT: $CI_COMMIT_TAG
+  only:
+    - tags
+
+# Not executed, parent job definition
+.deploy-python-tool-codon:
```

Comment:
True, I guess I was imagining the duplication would be temporary since the noah job would have to be removed anyway, and doing this would be an unnecessary layer of abstraction... on the other hand these things tend to hang around longer than anticipated so I see your point!